helmclient.go

```diff
@@ -253,6 +254,35 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 		}
 	}
 
+	// Create the priority class for tiller to make sure it is always deployed.
+
```

Comment:
Nitty but no need for a newline here IMO.

Reply:
Removed.